plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -621,4 +628,16 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public Image getImage() {
+    return image;
+  }
+
+  @Override
+  public void dispose() {
+    if (image != null)
```

Comment:
google style requires braces even for single line ifs

Reply:
Updated